contracts/distribution/ZNSEthRegistrar.sol

```diff
@@ -111,60 +108,56 @@ contract ZNSEthRegistrar is AccessControlled, IZNSEthRegistrar {
 
-    function reclaimDomain(bytes32 domainHash)
-    external
-    override
+    function reclaimDomain(bytes32 domainHash) external override
     onlyTokenOwner(domainHash)
     {
         registry.updateDomainOwner(domainHash, msg.sender);
 
         emit DomainReclaimed(domainHash, msg.sender);
     }
 
-    function setRegistry(address znsRegistry_) public override onlyAdmin {
+    function setRegistry(address registry_) public override onlyAdmin {
         require(
-            znsRegistry_ != address(0),
-            "ZNSEthRegistrar: znsRegistry_ is 0x0 address"
+            registry_ != address(0),
+            "ZNSEthRegistrar: registry_ is 0x0 address"
         );
-        registry = IZNSRegistry(znsRegistry_);
+        registry = IZNSRegistry(registry_);
 
-        emit RegistrySet(znsRegistry_);
+        emit RegistrySet(registry_);
     }
 
-    function setTreasury(address znsTreasury_) public override onlyAdmin {
+    function setTreasury(address treasury_) public override onlyAdmin {
         require(
-            znsTreasury_ != address(0),
-            "ZNSEthRegistrar: znsTreasury_ is 0x0 address"
+            treasury_ != address(0),
+            "ZNSEthRegistrar: treasury_ is 0x0 address"
         );
-        treasury = IZNSTreasury(znsTreasury_);
+        treasury = IZNSTreasury(treasury_);
 
-        emit TreasurySet(znsTreasury_);
+        emit TreasurySet(treasury_);
     }
 
-    function setDomainToken(address znsDomainToken_) public override onlyAdmin {
+    function setDomainToken(address domainToken_) public override onlyAdmin {
         require(
-            znsDomainToken_ != address(0),
-            "ZNSEthRegistrar: znsDomainToken_ is 0x0 address"
+            domainToken_ != address(0),
+            "ZNSEthRegistrar: domainToken_ is 0x0 address"
         );
-        domainToken = IZNSDomainToken(znsDomainToken_);
+        domainToken = IZNSDomainToken(domainToken_);
 
-        emit DomainTokenSet(znsDomainToken_);
+        emit DomainTokenSet(domainToken_);
     }
 
-    function setAddressResolver(address znsAddressResolver_) public override onlyAdmin {
+    function setAddressResolver(address addressResolver_) public override onlyAdmin {
         require(
-            znsAddressResolver_ != address(0),
-            "ZNSEthRegistrar: znsAddressResolver_ is 0x0 address"
+            addressResolver_ != address(0),
+            "ZNSEthRegistrar: addressResolver_ is 0x0 address"
         );
-        addressResolver = IZNSAddressResolver(znsAddressResolver_);
+        addressResolver = IZNSAddressResolver(addressResolver_);
 
-        emit AddressResolverSet(znsAddressResolver_);
+        emit AddressResolverSet(addressResolver_);
     }
 
-    function setAccessController(address accessController_)
-    external
-    override(AccessControlled, IZNSEthRegistrar)
-    onlyAdmin
+    function setAccessController(address accessController_) external override(AccessControlled, IZNSEthRegistrar) 
+        onlyAdmin
```

Comment:
again here. why did you bring the visibility keywords up, but not the modifier? what's wrong with the conventional approach that's already been there? also, why is modifier indented? this is "dirty syntax", let's revert back to how it was.